src/java.base/share/classes/java/util/HexDigits.java

```diff
@@ -39,27 +39,27 @@ final class HexDigits implements Digits {
      * Each element of the array represents the ascii encoded
      * hex relative to its index, for example:<p>
      * <pre>
-     *       0 -> '00' -> ('0' << 8) | '0' -> 12336
-     *       1 -> '01' -> ('0' << 8) | '1' -> 12337
-     *       2 -> '02' -> ('0' << 8) | '2' -> 12338
+     *       0 -> '00' -> ('0' << 8) | '0' -> 0x3030
+     *       1 -> '01' -> ('0' << 8) | '1' -> 0x3130
+     *       2 -> '02' -> ('0' << 8) | '2' -> 0x3230
```

Comment:
The description of the array content does not need to be exhaustive; just sufficient to understand how it is indexed and the meaning of the values.

Reply:
The comment is already there, I think the value on the right is easier to understand in hexadecimal

Reply:
comment has been removed